packages/fxa-settings/src/pages/Legal/index.tsx

```diff
@@ -20,7 +20,10 @@ const Legal = (_: RouteComponentProps) => {
       <CardHeader headingTextFtlId="legal-header" headingText="Legal" />
 
       <div className="flex justify-between text-sm">
-        <Link className="link-blue" to="/legal/terms">
+        <Link
+          className="link-blue ltr:mr-10 rtl:ml-10 mobileLandscape:mx-0"
```

Comment:
ignorant question: w/ tailwind, is it different/better to set bi-di `mr-10`/`ml-10` classes here versus something like `gap-x-10` above on the `div` on L22?

Reply:
Not an ignorant question at all, I didn't think about using `gap` for some reason (probably habit). It won't work for all cases, but here it's definitely much nicer so thanks! 😈